packages/frontend-2/components/header/NavUserMenu.vue

```diff
@@ -208,6 +215,22 @@ onMounted(() => {
       return
     }
 
+    if (workspaceQuery && isString(workspaceQuery)) {
+      workspaceSettingsDialogTarget.value = workspaceQuery
+
+      if (errorQuery && isString(errorQuery)) {
+        triggerNotification({
+          type: ToastNotificationType.Danger,
+          title: errorQuery
+        })
+      } else {
+        triggerNotification({
+          type: ToastNotificationType.Success,
+          title: 'Lekker bezig ah niffo'
```

Comment:
@cdriesler I wanted to change this ah niffo (🪦), but I'm not sure what is the correct message, is it something like "SSO settings successfully updated"?

Reply:
"SSO settings successfully updated" is good 👍